cirq-core/cirq/_compat.py

```diff
@@ -457,6 +457,7 @@ def __init__(
         new_module_name: str,
         old_module_name: str,
         deadline: str,
+        broken_module_exception: Optional[BaseException],
```

Comment:
type is optional but this argument isn't optional(?)

Reply:
I just marked it as it could be None...that doesn't mean you don't have to mandatorily state whether it's None or not...is that weird? I can make it None by default.

Reply:
I guess it's not that weird